lib/rucio/core/rse.py

```diff
@@ -1414,26 +1400,25 @@ def update_rse(rse_id: str, parameters: 'Dict[str, Any]', session=None):
         query = session.query(models.RSE).filter_by(id=rse_id).one()
     except sqlalchemy.orm.exc.NoResultFound:
         raise exception.RSENotFound('RSE with ID \'%s\' cannot be found' % rse_id)
-    availability = 0
     rse = query.rse
-    for column in query:
-        if column[0] == 'availability':
-            availability = column[1] or availability
+
```

Comment:
this looks like some ancient way to find a column. not entirely sure :confused:

Reply:
I think so too, just wanted to make sure.